### DISCUSSION TITLE
Maui Android performance

### DISCUSSION THREAD

**Discussion by jaintarun** [dotnet/maui#1881] [Q&A]
I came across this article written late last year. 

https://itnext.io/goodbye-xamarin-forms-f41723fb9fe1

The author mentions that xamarin.forms android performance issues will remain in Maui.
Any counterpoints?

**Comment by LuckyDucko** [comment]
With the use of [Microsoft.Maui.Graphics.Controls](https://github.com/dotnet/Microsoft.Maui.Graphics) and this talk of the new Thin Renderers to reduce the heft, Maui may have a counterpoint to the article which is a bit dated.
